### DISCUSSION TITLE
Module Blog. Add field for displaying copyright note of image, photo.

### DISCUSSION THREAD

**Discussion by Sellato** [forkcms/forkcms#3334] [Ideas / Feature Requests]
## Feature Request
I think it will be good to add to **Blog** module a field for displaying copyright note of image, photo.
Example in attach.

| Q           | A      |
| ----------- | ------ |
| New Feature | yes    |
| RFC         | yes |

![Screenshot_2019-11-07](https://user-images.githubusercontent.com/5456419/68373360-55c50c80-0175-11ea-84bc-7227a7fdce8c.png)

### Summary

Good practice is show the copyright notes of author of image, or type a license for using.

**Comment by carakas** [comment]
We are currently busy switching over Blog to the media library and doctrine. It would make more sence to me if we would add a copyright note in the Media Library

**Comment by Sellato** [comment]
At this time images loads to Blog directly from my PC, not from Media Library... So I don't know is it good or not to add copyright to MediaLibrary.

**Comment by carakas** [comment]
You can add it to your own website to blog directly, but we won't be accepting a PR for it since that module is currently being switched over to doctrine and media library.
A PR to add this to the media library would be something interesting for Fork, but I can understand that since you are currently using a version where the media library isn't used in the Blog module that that wouldn't solve your current problem

**Comment by Sellato** [comment]
:)
OK.
